modules/ocf_desktop/files/xsession/print-notify-listener

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -eu
+function clean_up {
+    jobs -pr | xargs pgrep -P | xargs sudo -u ocfbroker kill
+    exit 0
+}
+
+trap clean_up INT TERM EXIT
+
+pipe=$HOME/tmp/print-notify
```

Comment:
I think you should put the value here in quotes, not that `$HOME` should have spaces in it, but I guess it might.